app/models/solidus_subscriptions/subscription.rb

```diff
@@ -12,13 +12,15 @@ class Subscription < ActiveRecord::Base
     has_many :installments, class_name: 'SolidusSubscriptions::Installment'
     belongs_to :store, class_name: 'Spree::Store'
     belongs_to :shipping_address, class_name: 'Spree::Address'
+    belongs_to :payment_source, class_name: 'SolidusPaypalBraintree::Source'
```

Comment:
This caught my attention as it's making my test fail. If we want to do this, we probably need to link to a `wallet_payment_source` here instead of linking to a specific gateway's class.

Reply:
i'm pretty sure that's what the git history shows for how you got this working initially @fastjames so that makes sense to me

Reply:
Is that a decorator that adds `accept_nested_attributes_for :wallet_payment_source`? That seems like a reasonable thing to do.

Reply:
Yeah, trying to do that now. We shall see if I'm smart enough to make it work.